### PR TITLE
general updates: reflect status of March 2021

### PR DESCRIPTION
**Pull request by jbech-linaro**
- Clean-up the landing page and make it up-to-date.
- Add a line to clarify that the documentation is at readthedocs.
- Remove Linaro from the contact page.
- Add links to the TrustedFirmware.org news and blog post section. Also
  refer to the list of talks and presentations that can be found at one
  the documentation pages.

Signed-off-by: Joakim Bech <joakim.bech@linaro.org>

### CODE REVIEW

**Comment by jbech-linaro** [comment]
@kylekirkby  and @pcolmer I kindly ask for help figure out the checker-errors. I did one change trying to solve the line it's complaining about, but it still doesn't like it. Since I cannot run this locally, it's a bit cumbersome to do trial and error.

**Comment by pcolmer** [comment]
@jbech-linaro The error is this:

> YAML Exception reading /srv/source/_pages/latest.md: (<unknown>): found a tab character where an indentation space is expected while scanning a block scalar at line 15 column 17 

If you look at https://github.com/OP-TEE/optee_website/pull/131/files and scroll down to "latest.md", you'll see that the new lines are using tabs for indenting rather than spaces.

**Comment by jbech-linaro** [comment]
> If you look at https://github.com/OP-TEE/optee_website/pull/131/files and scroll down to "latest.md", you'll see that the new lines are using tabs for indenting rather than spaces.

Weird, I was reading exactly that and was looking for tabs at the line. Maybe I pushed the file at wrong time (saved, not saved, committed, not committed). Pushed once again.

File: _pages/homepage.md [hunk 1]
```diff
@@ -27,47 +27,34 @@ flow:
...
-            others. At the same time, the code base was cleaned up with the application of
-            [Coding
-            standards](https://optee.readthedocs.io/en/latest/general/coding_standards.html#coding-standards),
-            by running
-            [checkpatch](http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/scripts/checkpatch.pl),
-            and via other related processes.
+            working with STMicroelectronics to convert their TEE solution from being a
+            proprietary TEE to become an open source TEE. After a few months of
+            refactoring and rewriting major parts of the code so it was compatible with
+            the BSD 2-Clause license it was release to the general public around the
```
Comment (pcolmer) [inline _pages/homepage.md:45]: "released", not "release"

**Review by pcolmer — CHANGES_REQUESTED** [review]

**Comment by jbech-linaro** [comment]
One down, now I can see that it complains about a 404 for the Spectre & Meltdown blog post. Indeed it's gone and I cannot find it again. Do you have any clue what happened to it?

File: _pages/homepage.md (hunk 1 above)
Comment (jbech-linaro) [reply _pages/homepage.md:45]: Will fix, thanks!

**Comment by pcolmer** [comment]
@jbech-linaro 

> Do you have any clue what happened to it?

It got purged in a blog cleanup 😯

@kylekirkby is restoring it now.

**Comment by pcolmer** [comment]
@jbech-linaro Looking good now - the broken links are fixed.

**Comment by jbech-linaro** [comment]
> @jbech-linaro Looking good now - the broken links are fixed.

Yes, just noticed, hence I pushed again with the spelling error also fixed, thanks!

**Review by pcolmer — APPROVED** [review]

**Comment by jbech-linaro** [comment]
Thanks @pcolmer for the quick turn around time!

**Pull request merged**